examples/README.md

```diff
@@ -8,6 +8,12 @@ are intended to be run with. Note, however, that all cases
 in the Flow directory can be run with ECLIPSE, and _some_ of
 the cases in the ECLIPSE directory can be run with FLOW.
 
+Please note that the Flow simulator(2023.04) set GRIDFILE as 
```

Comment:
There are some problems in the text. I suggest the following corrected text (but please check if this is what you meant).

"Please notice that the OPM Flow simulator version 2023.04 uses the keyword GRIDFILE, which is only supported by OPM Flow. The optimization procedure performed by FieldOpt will not display this error in case the simulator is executed from the terminal. Please refer to the csv file in the output directory for more details."

Reply:
I believe it is only legacy reasons for including the GRIDFILE keyword. Flow uses the EGRID format only, so the GRIDFILE keyword is not needed in Flow. I therefore suggest removing this comment and the keyword altogether.